packages/bpk-tokens/tokens/base.raw.android.json

```diff
@@ -102,12 +102,6 @@
     "FONT_SIZE_XXL": {
       "value": 34
     },
-    "FONT_WEIGHT_BLACK": {
-      "value": "900"
-    },
-    "FONT_WEIGHT_BOLD": {
-      "value": "700"
-    },
```

Comment:
These were unused - aliases are not included in output (apart from raw but these token formats are not "advertised") so non breaking change.